cobra/core/model.py

```diff
@@ -316,6 +316,8 @@ def copy(self):
         except Exception:  # pragma: no cover
             new._solver = copy(self.solver)  # pragma: no cover
 
+        new._contexts = list()
```

Comment:
What is _contexts being used for? I don't see it being used anywhere else in the pull request.